serverless/components
# Auto-run ESLint and Prettier on every Git commit

## What has been implemented?

Automatically runs ESLint and Prettier on every Git commit, ensuring that only linted and prettified files are pushed.

Closes SC-244
Closes SC-245

## Steps to verify

Check out the branch. Create a new branch based on this branch. Modify a .js file (add ESLint violations and format it slightly). Stage the file and commit it. The pre commit checks should kick in --> ESLint and Prettier should auto-fix violations when possible.

## Todos:
